src/main/kotlin/calculator/StringAddCalculatorManager.kt

```diff
@@ -0,0 +1,41 @@
+package calculator
+
+import calculator.util.deleteCustomDelimiter
+import calculator.util.getCustomDelimiter
+
+class StringAddCalculatorManager {
+    fun start(input: String?): Int {
+        val parsedNumbers = inputToListOrThrow(input)
+        return StringAddCalculator(parsedNumbers).calculate()
+    }
+
+    fun inputToListOrThrow(input: String?): List<Int> {
+        if (input.isNullOrBlank()) return emptyList()
+        val parsedNumbers = split(input)
+        parsedNumbers.apply { if (any { it < 0 }) throw RuntimeException() }
+        return parsedNumbers
+    }
+
+    private fun split(input: String): List<Int> {
+        val customDelimiter: String = getCustomDelimiter(input)
```

Comment:
```suggestion
        val customDelimiter = getCustomDelimiter(input)
```
변수 명이 좋아서 반환 타입은 지워도 되겠네요~